docs/operator-scoping.md

```diff
@@ -0,0 +1,148 @@
+* **MultiNamespace**: If supported, the operator can be a member of an OperatorGroup that selects more than one namespace
+* **AllNamespaces**: If supported, the operator can be a member of an OperatorGroup that selects all namespaces (target namespace set is the empty string "")
+
+> Note: If a CSV's spec omits an entry of InstallModeType, that type is considered unsupported unless support can be inferred by an existing entry that implicitly supports it.
+
+Cluster admins cannot override which `InstallModes` an operator supports, and so should understand how to create an `OperatorGroup` that supports each `InstallMode`. Let's look at an example of an `OperatorGroup` implementing each type of installMode:
+
+##### OwnNamespace
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: own-namespace-operator-group
+  namespace: own-namespace
+spec:
+  targetNamespaces:
+  - own-namespace
+```
+
+##### SingleNamespace
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: single-namespace-operator-group
+  namespace: own-namespace
+spec:
+  targetNamespaces:
+  - some-other-namespace
+```
+
+##### MultiNamespace
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: multi-namespace-operator-group
+  namespace: own-namespace
+spec:
+  targetNamespaces:
+  - own-namespace
+  - some-other-namespace
+```
+
+##### AllNamespaces
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: all-namespaces-operator-group
+  namespace: own-namespace
+```
+
+### Scoping Member Operator Permissions
+
+When creating an `OperatorGroup`, cluster admins may specify a `ServiceAccount` that defines the set of permissions that may be granted to all member operators. OLM will ensure that when an operator is installed its privileges are confined to that of the `ServiceAccount` specified.
+
+As a result a cluster-admin can limit an operator to a pre-defined set of RBAC rules. The Operator will not be able to do anything that is not explicitly permitted by these permissions. This enables self-sufficient installation of Operators by non-cluster-admin users with a limited scope.
+
+#### Defining a ServiceAccount for an OperatorGroup
+
+A `ServiceAccount` may be defined within an `OperatorGroup` like so:
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: scoped-permissions-operator-group
+  namespace: own-namespace
+spec:
+  serviceAccountName: member-operator-servicee-account
+  targetNamespaces:
+  - own-namespace
+```
+
+Any operator tied to this OperatorGroup will now be confined to the permission(s) granted to the specified ServiceAccount. If the operator asks for permission(s) that are outside the scope of the ServiceAccount the install will fail with appropriate error(s). When OLM installs the operator the following will happen:
```

Comment:
It seems like there should be another section below this based on how the previous sentence ended. Or am I misreading?

Reply:
My mistake, the final sentence should not exist.